libraries/AP_HAL/examples/RCInput/RCInput.cpp

```diff
@@ -34,7 +34,7 @@ void loop(void)
         }
     }
     if (changed) {
-        for (uint8_t i=0; i<max_channels; i++) {
+        for (uint8_t i=0; i<=max_channels; i++) {
```

Comment:
In the current release code and tested with my 6 channel receiver:
- `nchannels` = 6
- First `for` loop: `i` goes from 0 to 5 (6 iterations)
- Therefore `max_channels` = 5
- Second `for` loop: `i` goes from 0 to 4 (1 less than `max_channels`) (only 5 iterations)
- Therefore channel 6 is not displayed. 

Alternatively, line 33 could be changed to `max_channels = i+1;` to ensure that `nchannels` and `max_channels` are both one-based for consistency.